chromadb/test/property/test_filtering.py

```diff
@@ -88,7 +88,10 @@ def _filter_embedding_set(recordset: strategies.RecordSet, filter: strategies.Fi
 
 
 collection_st = st.shared(
-    strategies.collections(add_filterable_data=True, with_hnsw_params=True), key="coll"
+    strategies.collections(
+        add_filterable_data=True, with_hnsw_params=True, has_embeddings=True
```

Comment:
Can we add a comment as to why the filter tests don't work with has_embeddings=True

Reply:
Refactored the test itself to no longer rely on this.